spec.bs

```diff
@@ -210,11 +210,9 @@ Requests to key commitment endpoints should result in a JSON response
 
 * `"batchsize"` specifies the maximum number of blinded tokens that the issuer
                 supports for each token issuance operation. Its value is a
-                string representation of a positive integer. Maximum value
-                allowed is 100. If a larger value is specified, 100 will be
-                used. Browser might send fewer tokens in a single operation,
-                but will generally default to sending `batchsize` many tokens
-                per operation.
+                string representation of a positive integer. Browser might send
```

Comment:
Maybe instead of "Browser" we can say "The user-agent"?

Reply:
I usually write it without a dash but +1